images/aquarium/config.xml

```diff
@@ -15,17 +15,19 @@
         <!-- Flavors -->
 	    <!-- <profile name="ContainerHost" description="Aquarium with container runtime (podman)"/> -->
         <!-- Platforms -->
-        <profile name="Vagrant-VirtualBox-x86_64" description="Vagrant for x86_64 VirtualBox" arch="x86_64"/>
-        <profile name="Vagrant-x86_64" description="Vagrant for x86_64 libvirt" arch="x86_64"/>
+        <profile name="Vagrant-VirtualBox-x86_64" description="Vagrant for
+        x86_64 with VirtualBox" arch="x86_64"/>
+        <profile name="Vagrant-x86_64" description="Vagrant for x86_64 with libvirt" arch="x86_64"/>
```

Comment:
I wonder if this change is really needed and if so, can put new lines between attributes instead of inside attribute values. Also it would be nice to have some indentation so the start of the tag is distinguished.

Reply:
It is not _needed_, but the wording was annoying me enough that I did it anyway. Good point with the wrapping and indenting.